rest/data/project_settings.go

```diff
@@ -116,6 +116,16 @@ func SaveProjectSettingsForSection(ctx context.Context, projectId string, change
 	modified := false
 	switch section {
 	case model.ProjectPageGeneralSection:
+		if mergedProjectRef.Identifier != mergedBeforeRef.Identifier {
```

Comment:
Are there any spruce changes needed as well for this ticket?

Reply:
Yup! There will be a PR adding an input to change the identifier on the general settings page.